iCubErzelli02/hardware/FT/left_arm-eb1-j0_3-strain.xml

```diff
@@ -16,16 +16,16 @@
             <group name="PROPERTIES">
   
                 <group name="CANBOARDS">
-                    <param name="type">                 eobrd_strain            </param>
+                    <param name="type">                 strain2            </param>
 
                     <group name="PROTOCOL">
-                        <param name="major">            1                       </param>    
+                        <param name="major">            2                       </param>    
                         <param name="minor">            0                       </param>     
                     </group>                    
                     <group name="FIRMWARE">
-                        <param name="major">            0                       </param>    
+                        <param name="major">            2                       </param>    
                         <param name="minor">            0                       </param> 
-                        <param name="build">            0                       </param>
+                        <param name="build">            9                       </param>
```

Comment:
What is this?
Any comment @davidetome ?

Reply:
An FT45 sensor (strain1) has been replaced with a strain2 sensor. After consulting with @marcoaccame we had to modify the configuration files for the version and protocol.